### DISCUSSION TITLE
Next release for .net 7 after 7.0.92

### DISCUSSION THREAD

**Discussion by Jake-Derrick** [dotnet/maui#16985] [General]
When can we expect the next release for .net7?

Mainly looking for the fix from this: https://github.com/dotnet/maui/pull/16651 

Would it be better to switch to .Net 8 instead of waiting for the next .Net 7 release?

**Comment by JohnHDev** [comment]
We have jumped to .NET 8, it was mentioned only 20% of fixes have been back ported to .NET 7.
We are looking to complete our migration in December, if there are no breaking changes/bugs we can't work around.

**Reply by developer9969** [reply]
@JohnHDev i would love to jump to net 8 but with so many regressions bugs and we also found out that resource dictionary-styles are broken when reading from a shared project- we are truly in a bad place with both 7 and 8 !!! What a mess

**Reply by RobTF** [reply]
Same here - .NET 8 regressions prevent us considering it yet.

**Reply by JohnHDev** [reply]
Im not aware of any specific regressions over .NET 7, although to be fair we haven't yet migrated anything other than a large test solution we created.
Fixes are going into .NET 8 first with only 20% ish being back ported to .NET 7. I strongly advise you migrate to .NET8 and report any and all bugs you find. More likely to be fixed in a timely manner.

**Reply by RobTF** [reply]
There's a fair few;

https://github.com/dotnet/maui/issues?q=is%3Aissue+is%3Aopen+label%3Ai%2Fregression

Most are titled with a `[regression/8.0.0]` prefix
